src/I18n/Number.php

```diff
@@ -276,6 +276,10 @@ public static function formatter($options = [])
             $type = NumberFormatter::CURRENCY;
         }
 
+        if (!empty($options['type']) && $options['type'] === 'ordinal') {
```

Comment:
Once can use any of the valid NumberFormatter [constants](http://in2.php.net/manual/en/class.numberformatter.php#intl.numberformatter-constants.unumberformatstyle) for the `type` option which are integers.